src/lib.rs

```diff
@@ -209,6 +210,37 @@ pub trait Device: AsRawFd {
 
         Ok(driver)
     }
+
+    /// Waits for a vblank.
+    fn wait_vblank(
+        &self,
+        frame: VblankWaitFrame,
+        flags: VblankWaitFlags,
+        high_crtc: u32,
+        user_data: usize,
+    ) -> Result<VblankWaitReply, SystemError> {
+        if high_crtc > 0x1f {
```

Comment:
Can the `_DRM_VBLANK_HIGH_CRTC_MASK` constant be used here, after shifted right by `_DRM_VBLANK_HIGH_CRTC_SHIFT`?